index.html

```diff
@@ -402,10 +402,15 @@ <h3 id="installability-signals">
           By design, this specification does not provide developers with an
           explicit API to "install" a web application. Instead, a
           <a>manifest</a> can serve as an <dfn>installability signal</dfn> to a
-          user agent that a web application can be installed.
+          user agent that a web application can be installed. These signals
+          will vary per user agent, as each user agent will have it's own
+          heuristics to determine whether a web site is elegible of an install
+          prompt. Developers should reference the relevant user-agent-specific
+          documentation to determine this per user agent.
```

Comment:
Just to avoid that "should" there, how about something like: 

```suggestion
          prompt. Implementers generally will provide documentation that describe their particular installabilty signals
          or other relevant criteria a web application needs to meet to be deemed installable.
```